src/components/Sources/config/FormField.tsx

```
@@ -0,0 +1,405 @@
+                {items.length === 0 && (
+                  <Box
+                    sx={{
+                      p: 2,
+                      textAlign: 'center',
+                      border: '1px dashed',
+                      borderColor: 'divider',
+                      borderRadius: 1,
+                    }}
+                  >
+                    <Typography variant="body2" color="text.secondary">
+                      No {field.title?.toLowerCase() || 'items'} configured. Click "Add" to create
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)